plutarch-docs/Tricks/Representation of Plutarch type.md

```diff
@@ -21,5 +21,5 @@ Now, let's discuss patterns of data declarations and what representation they _s
 - If it's a representationally equal wrapper (think Haskell `newtype`) to a term containing a Plutarch type - it should have the 
   same representation as that underlying Plutarch type.
 
-  e.g. `newtype PPubKeyHash (s :: S) = PPubKeyHash (Term s PByteString)` is just represented as `PByteString`. This is ensured 
+  e.g. `newtype PPubKeyHash (s :: S) = PPubKeyHash (Term s (PDataNewtype PByteString))` is just represented as `PByteString`. This is ensured 
```

Comment:
PPubkeyHash is not represented as `PByteString` as it is wrapped by `PDataNewtype`. 

It is better to give some new type 
like `newtype PMyType (s :: S) = PMyType (Term s PByteString)` which will actually have PByteString representation